devel/clavus/Portfile

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           rust 1.0
+
+name                clavus
+version             0.0.1
+categories          devel
+
+maintainers         {gmail.com:pietro.mobrici}
+
+platforms           darwin
+supported_archs     x86_64
```

Comment:
does this really only work on `x86_64` or can the default MacPorts setting be used?